src/private/files.ts

```diff
@@ -226,6 +226,43 @@ export namespace files {
     listUrl?: string;
   }
 
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Download Files interface
+   */
+  export interface IFileItem {
+    /**
+     * ID of the file metadata
+     */
+    objectId?: string;
+    /**
+     * Path of the file
+     */
+    path?: string;
+    /**
+     * Size of the file
+     */
+    size?: number;
+    /**
+     * Download status
+     */
+    status?: string;
+    /**
+     * Download timestamp
+     */
+    timestamp: Date | string;
```

Comment:
>string [](http://example.com/codeflow?start=22&length=6)

Why string? And what format would the string be in?

Reply:
timestamp is represented as Date or string to remain consistent with the underlying API output data structure format. The string will also be a **Date()** object representation.

Reply:
I don't think we should leak the underlying implementation details of the system that's providing this data. If this is a string, how would anybody know what format this string is in?

Reply:
Also, if it's a timestamp, shouldn't it be a `number` to begin with?

Reply:
Name of the filed is **timestamp** but it is represented by Date object.

Reply:
ok. In that case, it should be typed as `Date`, and not `Date | string`.

Reply:
Suggestion incorporated.